api/v1alpha1/ratelimit_types.go

```diff
@@ -62,6 +62,7 @@ type LocalRateLimit struct {
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
+	// +kubebuilder:validation:XValidation:rule="self.all(foo, !has(foo.cost.response))", message="response cost is not supported for Local Rate Limits"
```

Comment:
@mathetake is this the cause of failed tests?

Reply:
huh